arrpy/random.py

```
@@ -0,0 +1,97 @@
+"""
+
+import random as _random
+
+
+class RandomState:
+    """Random number generator state."""
+    
+    def __init__(self, seed=None):
+        """
+        Initialize random state.
+        
```

Ruff (0.12.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)